src/Driver/Connection.php

```diff
@@ -64,7 +64,7 @@ public function lastInsertId($name = null);
      *
      * @throws Exception
      */
-    public function beginTransaction();
+    public function beginTransaction(): bool;
```

Comment:
This is a BC-break for implementations.

Reply:
Ah so we cannot add it now? I though it's not since it was already declared in docblock but I understand implementations missing this will not compile.

Reply:
Also, it has already been done on 4.0.x: https://github.com/doctrine/dbal/blob/06243b707eee0d108e7306f830a36d70a0206f85/src/Driver/Connection.php#L69 (note that it should be `void` BTW

Reply:
Ok I guess I'll have to work around this then.